src/components/AddTodo.js

```diff
@@ -0,0 +1,38 @@
+import Component from './Component.js';
+
+class AddTodo extends Component {
+
+    render() {
+        const form = this.renderDOM();
+        const onAdd = this.props.onAdd;
+
+        form.addEventListener('submit', event => {
+            event.preventDefault();
+
+            const formData = new FormData(form);
+            const newTodo = {
+                task: formData.get('link'),
+                completed: formData.get('completed')
+            };
+
+            onAdd(newTodo);
+
+            form.reset();
+            document.activeElement.blur();
+        });
+
+        return form;
+    }
+
+    renderTemplate() {
+        return /*html*/`
+            <form>
+                <label>Add To-Do
+                    <input type="text" name="link">
+                <label>
+            <form>
```

Comment:
You're so close! It looks like you have all the functionality set up to add your new todos but I don't see a submit button which means the form is never submitted and the list isn't updated.